nemo/collections/asr/parts/utils/vad_utils.py

```diff
@@ -637,7 +637,7 @@ def generate_vad_segment_table_per_tensor(sequence: torch.Tensor, per_args: Dict
 
     speech_segments, _ = torch.sort(speech_segments, 0)
 
-    dur = speech_segments[:, 1:2] - speech_segments[:, 0:1] + shift_length_in_sec
+    dur = speech_segments[:, 1:2] - speech_segments[:, 0:1] + 0.01  # 10ms frame unit
```

Comment:
How about adding this variable as  something like "segment margin" in per args?
Similar thing is happening in diarization and it is parameterized.

Reply:
I was struggling to find a good name for the fixed universal 10ms frame len. I don't need it to be parameterized and it used to be frame_len but we changed them all. Any suggestion? how about FRAME_UNIT=0.01?

Reply:
In that seense "UNIT_FRAME_LEN" or "UNIT_FRAME" will be better because this is "length" not a "unit"

Reply:
great! Thanks!